odoo/addons/base/models/ir_sequence.py

```diff
@@ -196,6 +197,19 @@ def write(self, values):
         self.flush_model(values.keys())
         return res
 
+    @api.depends('use_date_range', 'date_range_ids', 'number_next_actual', 'date_range_ids.date_from', 'date_range_ids.date_to', 'date_range_ids.number_next_actual')
+    def _compute_sequence_preview(self):
```

Comment:
IMO this should depend on "prefix" and "suffix", that way if you modify those fields the preview is correctly updated.